reliability-v2/tasks/Tasks.py

```diff
@@ -443,4 +445,42 @@ def check_netobserv_pods(self, user):
             else:
                 self.logger.error(f"Pods status fetch error: result {result}, rc - {rc}")
                 rc_return = 1
-        return (result, rc_return)
\ No newline at end of file
+        self.__log_result(rc)
+        return (result, rc_return)
+
+    # Get all pods in a namespace
+    def get_all_pods_in_namespace(self, user, namespace):
+        pod_names = {}
+        self.logger.info(f"[Task] User {user}: get all pods in a namespace")
+        # Get all pods in a particular namespace
+        (result, rc) = oc(
+            "get pods -n {namespace} -o jsonpath='{.items[*].metadata.name}'",
+            self.__get_kubeconfig(user),
+            ignore_log=True,
+            ignore_slack=True,
+        )
+        if rc == 0 and result != "":
+            self.logger.info(f"Pods in ns {namespace} are: {result}")  
+            pod_names = string.Split(string.Trim(result, "'"), " ")
+        return pod_names
+    
+    # check netobserv pod restarts
+    def check_netobserv_pod_restarts(self, user):
+        self.logger.info(f"[Task] User {user}: check pod restarts")
+        # Check if pods have restarted
+        for ns in ("netobserv", "netobserv-privileged", "openshift-netobserv-operator"):
```

Comment:
Another option is to make the namespaces as a string for -p parameters, like apply_nonamespace(self,user,parameter). So this function can be `def check_pod_restarts(self, user, parameter)`.
Pros: it will be more common
Cons: in logs and notifications, the task name does not include netobserv